README.md

```diff
@@ -75,6 +80,7 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 | :------ | :---------- | :-------- |
 | [Critical Manufacturing](https://www.criticalmanufacturing.com) [:rocket:](https://www.criticalmanufacturing.com/en/careers) | Manufacturing business-critical applications. | `Porto` |
 | [Hitachi Vantara](https://www.hitachivantara.com/) [:rocket:](https://www.hitachivantara.com/en-us/company/careers.html) | IoT, cloud, application, big data and analytics solutions. | `Oeiras` `Porto` |
+| [PHC Software](http://www.phcsoftware.com/) [:rocket:](https://www.phcsoftware.com/carreiras/) | ERP software. | `Lisboa` `Porto` |
```

Comment:
Travis is reporting that the following link is return a `HTTP 500` error:
http://www.phcsoftware.com/

I have checked and the same one with HTTPS seems to work OK.  
Can you please check and update it?

Thanks.

Reply:
Done. I had actually hit it with https but typed it wrong.